src/components/CommentBox.spec.js

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import CommentBox from './CommentBox';
+
+describe('CommentBox', () => {
+  it('renders a heading, a textarea and a button', () => {
+    render(<CommentBox />);
+
+    const heading = screen.getByText(/leave/i);
+    expect(heading).toBeInTheDocument();
+
+    const textarea = screen.getByPlaceholderText('Add a comment...');
+    expect(textarea).toBeInTheDocument();
+
+    const submitButton = screen.getByRole('button');
+    expect(submitButton).toBeInTheDocument();
+  });
+
+  it('when the drink has comments in its data, a li element should be rendered', () => {
+    render(
+      <CommentBox
+        comments={[{ commentId: '123', commentText: 'dies ist ein Comment' }]}
+      />
+    );
+
+    const commentItem = screen.getByRole('listitem');
+    expect(commentItem).toBeInTheDocument();
+  });
```

Comment:
I like the test idea, but I think it would be better to check if the `commentText` is displayed if the drink has a comment in its data. 🍹 And it would be better, if your test data represents more a "real" comment a user would type.

Reply:
I will change this with a more real data